### PR TITLE
Perk countdown in magic trait tooltip

### LINKED ISSUE #595: Change tooltip in magic lifestyles about traits
**Describe your suggestion in full detail below:**

@ValianBlue suggestion

> I would personally change it to "Remaining [Magic] Perks to get [Level]: [X]." In other words, have it count down with each perk the player acquires as opposed to listing the threshold.

### PR DESCRIPTION
**Pull request by MemeWizard**
#### Changelog:
- Perks necessary for next level of magical trait in the GUI of magical lifestyle are now displayed as `№ of current/№ of needed`
- Close #595, because it's no longer needed

### CODE REVIEW

**Review by ValianBlue — APPROVED** [review]
_Works like a charm. Nice work!_

File: common/customizable_localization/wc_magical_lifestyles_custom_loc.txt [hunk 1]
```diff
@@ -386,4 +386,345 @@ GetWCMagicalLifestyleElementalEarthMagicNextTraitForPerks = {
 
 		localization_key = "wc_elemental_earth_magic_next_trait_level_is_1"
 	}
+}
+
+GetWCMagicalLifestyleLightHowMuchPerksForNextTrait = {
```
Comment (zumbak04) [inline common/customizable_localization/wc_magical_lifestyles_custom_loc.txt:391]: Keep only one way. Both numbers tell the same info.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/customizable_localization/wc_magical_lifestyles_custom_loc.txt (hunk 1 above)
Comment (MemeWizard) [reply common/customizable_localization/wc_magical_lifestyles_custom_loc.txt:391]: Well, no countdown then..

**Review by ValianBlue — APPROVED** [review]
_I think it would be best to close this issue as there's no net change, but I leave that to your discretion._

File: localization/english/gui/wc_magic_lifestyles_window_character_lifestyle_l_english.yml [hunk 2]
```diff
@@ -4,82 +4,88 @@
...
+ wc_elemental_earth_magic_next_trait_level_is_3:0 "required to get [GetTrait('user_of_elemental_earth_magic_3').GetName( GetPlayer )]"
+ wc_elemental_earth_magic_next_trait_level_is_4:0 "required to get [GetTrait('user_of_elemental_earth_magic_4').GetName( GetPlayer )]"
+ wc_elemental_earth_magic_next_trait_level_is_5:0 "required to get [GetTrait('user_of_elemental_earth_magic_5').GetName( GetPlayer )]"
  wc_elemental_earth_magic_next_trait_level_is_last:0 "make [GetPlayer.GetFirstName] a [GetTrait('user_of_elemental_earth_magic_5').GetName( GetPlayer )]"
+
+ wc_magic_next_trait_level_is_1_perk_amount:0 "[EmptyScope.ScriptValue('wc_perks_needed_for_level_1_magic_trait_value')|0]"
+ wc_magic_next_trait_level_is_2_perk_amount:0 "[EmptyScope.ScriptValue('wc_perks_needed_for_level_2_magic_trait_value')|0]"
+ wc_magic_next_trait_level_is_3_perk_amount:0 "[EmptyScope.ScriptValue('wc_perks_needed_for_level_3_magic_trait_value')|0]"
+ wc_magic_next_trait_level_is_4_perk_amount:0 "[EmptyScope.ScriptValue('wc_perks_needed_for_level_4_magic_trait_value')|0]"
+ wc_magic_next_trait_level_is_last_perk_amount:0 "[EmptyScope.ScriptValue('wc_perks_needed_for_level_5_magic_trait_value')|0]"
```
Comment (zumbak04) [inline localization/english/gui/wc_magic_lifestyles_window_character_lifestyle_l_english.yml:91]: There's no text, only value so you could make it without using one script value. Do you want to remake it?

Comment (MemeWizard) [reply localization/english/gui/wc_magic_lifestyles_window_character_lifestyle_l_english.yml]: Done

**Review by ValianBlue — APPROVED** [review]
_No issues found while testing._

**Review by zumbak04 — APPROVED** [review]
LGTM! Good job!

**Comment by MemeWizard** [comment]
@ValianBlue @zumbak04 thanks for reviews <3

**Pull request merged**